### PR TITLE
mgmt, webpubsub, add live tests

### PR DESCRIPTION
**Pull request by v-hongli1**
relate to https://github.com/Azure/azure-sdk-for-java/issues/35316

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    public void testCreateWebPubSubResource() {
+        WebPubSubResource webPubSubResource = null;
+        try {
+            String resourceName = "webpubsub" + randomPadding();
+            // @embedmeStart
+            webPubSubResource = webPubSubManager.webPubSubs()
+                .define(resourceName)
+                .withRegion(REGION)
+                .withExistingResourceGroup(resourceGroupName)
+                .withSku(new ResourceSku().withName("Premium_P1").withCapacity(1).withTier(WebPubSubSkuTier.PREMIUM))
```
Comment (weidongxu-microsoft) [inline sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java:89]: can we use a low cost tier?

it should have a free tier.

File: sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                .withTls(new WebPubSubTlsSettings().withClientCertEnabled(false))
+                .withNetworkACLs(new WebPubSubNetworkACLs()
+                    .withDefaultAction(AclAction.DENY)
+                    .withPublicNetwork(new NetworkAcl().withAllow(
+                        Arrays.asList(WebPubSubRequestType.SERVER_CONNECTION,
+                            WebPubSubRequestType.CLIENT_CONNECTION,
+                            WebPubSubRequestType.RESTAPI,
+                            WebPubSubRequestType.TRACE)
+                    ))
+                    .withPrivateEndpoints(Collections.emptyList())
```
Comment (weidongxu-microsoft) [inline sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java:99]: remove if not needed

Comment (v-hongli1) [reply sdk/webpubsub/azure-resourcemanager-webpubsub/src/test/java/com/azure/resourcemanager/webpubsub/WebPubSubManagerTests.java]: Redundant parameter settings have been removed and `Sku` type has been changed to `Free` in the new version.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**